config/opal_load_platform.m4

```diff
@@ -97,6 +97,27 @@ AC_DEFUN([OPAL_LOAD_PLATFORM], [
             AC_SUBST(OPAL_PARAM_FROM_PLATFORM, "no")
         fi
 
+        patch_dir="${with_platform}.patches"
+        patch_done="${srcdir}/.platform_patches"
+        if [ test -d "${patch_dir}" ]; then
```

Comment:
Isn't "[ test" redundant?  I.e., don't you just need "if [ ...]" or "if test ..."?